var/spack/repos/builtin/packages/hepmc3/package.py

```diff
@@ -61,4 +60,5 @@ def cmake_args(self):
 
         if self.spec.satisfies('+rootio'):
             args.append('-DROOT_DIR={0}'.format(self.spec['root'].prefix))
+        args.append("-DHEPMC3_ENABLE_TEST={0}".format(self.run_tests))
```

Comment:
This is a good opportunity to transition to the cmake helper functions (which add the `BOOL` type for boolean python values and join lists with `;`):
```suggestion
        args.append(self.define('HEPMC3_ENABLE_TEST', self.run_tests))
```

Reply:
Sure, I'll update those.

Reply:
You can just worry about it for next time if the opportunity arises -- this one's pipeline has passed so I'll just merge it. Thanks!